src/gui/qgsmaptool.cpp

```diff
@@ -66,7 +66,7 @@ QgsPointXY QgsMapTool::toMapCoordinates( const QgsMapLayer *layer, const QgsPoin
 
 QgsRectangle QgsMapTool::toLayerCoordinates( const QgsMapLayer *layer, const QgsRectangle &rect )
 {
-  return mCanvas->mapSettings().mapToLayerCoordinates( layer, rect );
+  return mCanvas->mapSettings().outputExtentToLayerExtent( layer, rect );
```

Comment:
I was not sure whether I can just change this to `outputExtentToLayerExtent` or whether I should introduce a new method in `QgsMapTool` and call that method from `QgsMapToolIdentify::identifyVectorLayer`.

Comments on that would be appreciated.